### PR TITLE
Add image for question & quiz

### PR DESCRIPTION
**Pull request by horiaradu**
* DragAndDrop file input
* handle image / image url state
* call API

### CODE REVIEW

**Comment by horiaradu** [comment]
I didn't find how to:

* edit the photo of a question (edit the question actually)
* choose / edit a quiz's photo

File: src/components/DragAndDrop.module.css [hunk 1]
```diff
@@ -18,10 +18,29 @@
...
+  right: 0;
+  width: 100%;
+  height: 100%;
+  border: 0;
+  z-index: 777;
+  opacity: 0;
+}
+
+.image {
+  max-width: 100%;
```
Comment (PaulMacinic) [inline src/components/DragAndDrop.module.css:44]: Image looks a bit weird, change `.image` to:
.image {
    position: absolute;
    height: auto;
    width: 100%;
    max-height: 100%;
    top: 0;
    left: 0;
 }

File: src/screens/CreateQuiz.js [hunk 2]
```diff
@@ -253,7 +272,15 @@ const CreateQuiz = () => {
                   </div>
 
                   <div className={classNames("box-two-thirds", styles.dnd)}>
-                    <DragAndDrop label={"Question image"} />
+                    <DragAndDrop
+                      label={"Question image"}
+                      onFileUpload={onQuestionImageUpload}
+                      imageUrl={
+                        activeQuestion
```
Comment (PaulMacinic) [inline src/screens/CreateQuiz.js:279]: We need to handle here the cases where a quiz has no image. It's not mandatory for a quiz to have an image, In that case we should not show any image here.

**Review by PaulMacinic — CHANGES_REQUESTED** [review]
See comments

File: src/screens/CreateQuiz.js (hunk 2 above)
Comment (horiaradu) [reply src/screens/CreateQuiz.js:279]: at this point, if it has no image, i will show the empty drag and drop area

File: src/components/DragAndDrop.module.css (hunk 1 above)
Comment (horiaradu) [reply src/components/DragAndDrop.module.css:44]: this breaks the aspect ratio of the image

**Pull request merged**